specification/data-resource-semantic-conventions.md

```diff
@@ -70,6 +70,28 @@ Labels defining a compute unit (e.g. Container, Process, Lambda Function).
 | container.image.name | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` |
 | container.image.tag | Container image tag. | `0.1` |
 
+
+### Function as a Service
+
+**type:** `faas`
+
+**Description:** A serverless instance.
+
+| Label  | Description  | Example  | Required |
+|---|---|---|--|
+| faas.region| A specific geographical location where the function is executed. | `us-east-2` | Yes |
+| faas.name | The name of the function being executed. | `opentelemetry-lambda` | Yes |
+| faas.identifier | The unique name of the function being executed. <br /> For example, in AWS this field correspond to the [ARN] value, in GCP to the URI of the resource, and Azure to the [FunctionDirectory] field. | `opentelemetry-lambda` | Yes |
+| faas.version | The version of the function being executed. | `opentelemetry-latest` | No |
+| faas.provider | Provider name of the function, e.g. `aws`, `gpc`, `azure`, `serverless`, ... | `aws` | No |
```

Comment:
What do you do if AWS has a second FaaS service which competes with Lambda? Azure already has multiple different types of function service. Is this meant to be the name of the Platform, or of the service used?

Reply:
I don't think this should be optional.

Reply:
My idea was to cover the platform, but from an end-user perspective maybe the name of the function service is better. This way, end-users can aggregate trace data by function service and not just by platform.  WDYT?

Reply:
This smells like it could be more general like, `cloud.provider=aws`, `cloud.service=lambda`.

Reply:
`gpc` -> `gcp`?